rootfs/conf/kops/helmfile.yaml

```diff
@@ -42,6 +42,7 @@ releases:
     chart: "chartmuseum"
     component: "platform"
     namespace: "kube-system"
+    vendor: "kubernetes"
```

Comment:
Hrmm doesn't seem like the appropriate vendor name. It shouldn't be the org associated with the chart (E.g. `kubenetes`), but the org associated with the github repo where the app is stored.

e.g. `https://github.com/cloudposse/geodesic`, means the vendor is `cloudposse`